README.md

```diff
@@ -1,2 +1,36 @@
 # COMP1500StarterPack
-Starter pack for COMP1500
+Standardized folder structure for COMP 1500: Intro to Programming with C# class at POCU
+
+## Mandatory Tools
+These tools are required to take this course
+1. Slack
+1. A Git client. (SourceTree recommended)
+
+## Recommended OS and Tools
+The following environment is recommended to take this course. If you do not want to use Windows, see Alternative Environment and Tools section below. However, all course instructions are made with Visual Studio in mind, so if you choose to use another environment and tools, you will need to Google how to setup your environment: it is not hard, just more work for you.
+1. Windows OS
+1. Visual Studio 2019 (NOT Visual Studio Code)
+	1. **IMPORTANT**: Select `.NET desktop development` during installation
+	1. Any edition is fine.
+
+## Alternative OS and Tools
+Alternatively, you can choose any environment and tools you want to develop and debug your labs and assignments locally. However, the grading will still be done via Visual Studio. If you stick to C# standards covered in the class, there won't be any problem. If that's not the case, you are responsible to fix any warnings and errors reported by the grading system.
```

Comment:
Alternatively, you can choose any environment and tools to debug your labs and assignments locally.